### DISCUSSION TITLE
Anyway to automate a reinterview?

### DISCUSSION THREAD

**Discussion by nervling** [zwave-js/zwave-js-ui#1848] [Q&A]
Is there any way to trigger an automated re-interview of a failed node? I have one zwave device that sometimes fails on startup, but re-interviewing from the UI quickly fixes that. Is there a way I can trigger auto retries? Or perhaps integrate an automation to do this in Nodered?

**Comment by robertsLando** [comment]
@nervling I think that fixing the fail is a better approach then doing an automatic re-interview. Could you add some logs of startup please?

BTW to automate the interview you can use mqtt apis

**Comment by nervling** [comment]
I'll have to see if I can catch it happening again. I suspect it's more of a hardware issue though, it's a bit further relative to the other nodes in my network, and is enclosed a bit, so it's possibly caused by a weaker signal. If there's a way to make it more tolerant of that I would prefer that approach, but I didn't see any obvious configuration options. Re-interview is certainly hacky, but seemed like it would work.
